app/i18n/locales/zh-Hant.json

```diff
@@ -413,9 +413,6 @@
   "wallet.settings.changePassword.dialog.repeatPasswordFieldPlaceholder": "重複新密碼",
   "wallet.settings.changePassword.dialog.repeatPasswordLabel": "重複密碼",
   "wallet.settings.changePassword.dialog.title.changePassword": "變更密碼",
-  "wallet.settings.changePassword.dialog.title.setPassword": "密碼",
```

Comment:
Please don't edit non-EN files. It causes terrible merge conflicts

Reply:
@SebastienGllmt 
Even keys has been delete ?

Reply:
Yes. Crowdin will delete them automatically for you